library/cwm/src/lib/cwm/abstract_widget.rb

```diff
@@ -145,6 +154,8 @@ def cwm_definition
       res["_cwm_key"] = widget_id
       if respond_to?(:help)
         res["help"] = help
+      elsif respond_to?(:dynamic_help)
```

Comment:
I am still not decided if I want `AbstractWidget#help` to be polymorphic ("string" or "string ()") but it would be an easy change here: (plus deleting dynamic_help_method)

```rb
  if respond_to?(:help)
    h = help
    if h.respond_to? :call
      h = fun_ref(method(:help), "string ()")
    end
    res["help"] = h
  else
    res["no_help"] = ""
  end
```
```